src/cascadia/TerminalControl/TermControlAutomationPeer.h

```diff
@@ -81,5 +82,6 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         winrt::weak_ref<Microsoft::Terminal::Control::implementation::TermControl> _termControl;
         Control::InteractivityAutomationPeer _contentAutomationPeer;
         til::shared_mutex<std::deque<wchar_t>> _keyEvents;
+        bool _closing;

```

Comment:
big whoa! make sure you initialize this - it will otherwise contain random data and some subset of users will report that they never get text notifications any more!